src/server/stream_family_test.cc

```diff
@@ -499,4 +499,54 @@ TEST_F(StreamFamilyTest, XTrimInvalidArgs) {
   EXPECT_THAT(resp, ErrArg("syntax error"));
 }
 
+TEST_F(StreamFamilyTest, Xclaim) {
+  Run({"xadd", "foo", "1-0", "k1", "v1"});
+  Run({"xadd", "foo", "1-1", "k2", "v2"});
+  Run({"xadd", "foo", "1-2", "k3", "v3"});
+  Run({"xadd", "foo", "1-3", "k4", "v4"});
+
+  // create a group for foo stream
+  Run({"xgroup", "create", "foo", "group", "0"});
+  // alice consume all the stream entries
+  Run({"xreadgroup", "group", "group", "alice", "streams", "foo", ">"});
+
+  // bob claims alice's two pending stream entries
+  auto resp = Run({"xclaim", "foo", "group", "bob", "0", "1-2", "1-3"});
+  EXPECT_THAT(resp, RespArray(ElementsAre(RespArray(ElementsAre("1-2", ArrLen(2))),
+                                          RespArray(ElementsAre("1-3", ArrLen(2))))));
+
+  // bob really have these claimed entries
+  resp = Run({"xreadgroup", "group", "group", "bob", "streams", "foo", "0"});
+  EXPECT_THAT(resp, ArrLen(2));
+  EXPECT_THAT(resp.GetVec()[1].GetVec()[0].GetVec(), ElementsAre("1-2", ArrLen(2)));
+  EXPECT_THAT(resp.GetVec()[1].GetVec()[1].GetVec(), ElementsAre("1-3", ArrLen(2)));
```

Comment:
Here and below (in many places in this test), please use `RespArray()` like you did above. It's more readable and also has better coverage